source/containers.tex

```diff
@@ -9300,23 +9300,25 @@
 \tcode{vector<bool, Alloc>} is not a program-defined specialization.
 \end{itemdescr}
 
-% FIXME: Where are these codeblocks supposed to be part of?
 \indexlibraryglobal{formatter}%
 \begin{codeblock}
-template<class T, class charT> requires is-vector-bool-reference<T>
-  struct formatter<T, charT> {
-  private:
-    formatter<bool, charT> @\exposid{underlying_}@;       // \expos
+namespace std {
+  template<class T, class charT>
+      requires @\exposid{is-vector-bool-reference}@<T>
```

Comment:
Lines 9307-9320 need to lose 2 spaces of indentation.

Reply:
fixed